tests/runtime/test_command_buffer_multi_device.c

```diff
@@ -303,7 +303,7 @@ main (int _argc, char **_argv)
       CHECK_CL_ERROR (clReleaseEvent (command_buf_event));
     }
 
-  /* Remap from 2 to 1 queues & run */
+  /* Remap from N to 1 queues & run */
```

Comment:
Can we test the opposite case as well? That is, the one that can enable runtime-driven task mapping/perf. portability via pushing all work to a default CPU queue and then remapping to utilize more devices. That likely is a NOP for now. Also it would be good to test a case where the remap is done to queue/devices which are not in the original schedule/buffer. This models the disappearing devices case.